### DISCUSSION TITLE
Trabalhar remoto ou alocado?

### DISCUSSION THREAD

**Discussion by felipefialho** [frontendbr/forum#67] [Discussão]
Agora já faz mais de 1 ano que estou trabalhando de casa, mas em meados do ano passado escrevi esse texto sobre [trabalhar remoto](http://www.felipefialho.com/blog/2015/sobre-trabalhar-remoto).

Separando em tópicos algumas vantagens e desvantagens, colocaria assim:

**Vantagens**
- Melhor qualidade de vida
- Menos tempo perdido no trânsito, etc e consequentemente [aumento do valor hora](http://www.felipefialho.com/blog/2015/uma-reflexao-sobre-salarios-valor-hora-e-qualidade-de-vida)
- Mais tempo para estudar e fazer cursos
- Maior flexibilidade de horários
- Mais tranquilidade para focar no trabalho (para alguns pode ser uma desvantagem) 
- Poder trabalhar de qualquer lugar, necessitando apenas do notebook e de internet

**Desvantagens**
- Menos contato com outros profissionais
- Pouco contato com outras pessoas no dia-a-dia
- Mais possibilidade de perder o foco
- Necessita muita organização com horários

---

Acredito que boa parte das desvantagens podem ser contornadas, como participar de meetups, eventos e forums para se manter atualizado, trabalhar em locais "públicos" para ter contato com outras pessoas, e se esforçar para manter os horários e o foco nas tarefas.

O maior problema é que a evolução técnica é mais lenta do que quando você esta em contato direto com desenvolvedores de alto nível, apesar de conversarmos muito por Skype, a dinâmica não é igual quando está do lado. 

E vocês, o que preferem?

**Comment by bernardodiasc** [comment]
Cara, sou suspeito pra falar, trabalho remoto ta no meu sangue, demiti todos meus patrões hehehe

Os prós e contras que eu listaria são exatamente iguais aos que voce listou @LFeh . Trabalhar remoto exige que o profissional seja autônomo, responsável e muito proativo. São qualidades de qualquer profissional, claro, mas meio que obrigatório para trabalhar remoto. Os desafios são grandes especialmente no começo, mas os pros são muito maiores que os contras, os contras podem ser transformados em pros com o ganho de experiência e disciplina e com simples ações como voce ja comentou, sair da zona de conforto (to nesse estágio atualmente).

O melhor de trabalhar remoto é poder priorizar a própria vida antes da vida da empresa. Digo, meu horário minhas regras minha vida. Tive muita sorte de entrar pra uma empresa que simplesmente valoriza isso e me apoia onde estou, eles dizem que o meu caminho é esse que está na minha frente, e é por ele que vou trilhar meu desenvolvimento profissional (e pessoal). Inclusive sinto que tenho um apoio muito grande por gostar de morar no Brasil, nada contra quem quer fugir daqui, entendo perfeitamente, sei das tretas, mas recebo valor moral por estar tentando fazer a diferença aqui, tentando melhorar as coisas aqui.

De maneira alguma estou criticando outras opiniões, que fique claro. Isso é bem pessoal, cada pessoa se identifica de forma diferente no universo. As vezes sinto muita falta de trabalhar do lado de outras pessoas dentro de uma empresa tendo objetivos em comum com essas outras pessoas, pois é diferente de trabalhar num coworking, por exemplo, onde cada um ta fazendo suas coisas isoladas :)

**Comment by felipefialho** [comment]
Acabei de ler mais um [ótimo texto sobre o assunto](https://medium.com/@interney/home-office-na-vida-profissional-e-pessoal-uma-moderniza%C3%A7%C3%A3o-necess%C3%A1ria-c38b2a2a2de6#.9tx9qldfi) no Medium.

**Comment by ribeiroevandro** [comment]
O pior desse tipo de trabalho é realmente ficar sozinho, as vezes bate uma bad maneira. Num trampo que fiz para black friday a equipe ficou o dia todo online no hangout, alguns deixavam o mic no mute, mas sempre estavamos conversando e trabalhando, foi uma forma bacana de deixar um ar de escritório mesmo estando de casa.

**Comment by rickbenetti** [comment]
Faz 2 anos que saí da empresa onde eu estava para eles virarem meus clientes, sempre quis trabalhar de casa, sempre rendi mais aqui do que num local cheio de telefone tocando, gente gritando, busão lotado e muito mais.

Só não estou ainda bem, pois não alcancei clientes suficientes com fee suficiente, hoje pago as despesas da empresa e male male sobra dinheiro pra pegar busão, e como não tenho level algum de javascript achar trampo de designer + frontend com salário descente tá difícil eu continuo tentando como freelancer e como empresa.

Ainda não alcancei o status de poder me sustentar sem depender da familia, alias trabalhar com Internet nunca me sustentou, mas continuo tentando.

---

Sobre rotinas, estou tentando uma nova, antes eu ia dormir as 05h, e às 14h tava online, agora eu conecto às 07h, me desconecto à 18h (ao menos tento), e ainda estou buscando separar a parte da manhã dia sim dia não para aprender javascript (20 anos de web sem javascript num dá mais rs).

**Comment by felipefialho** [comment]
Realmente @rkb81, o mercado está cada vez mais dificil para desenvolvedores que não sabem pelo menos o básico de JavaScript, tenta focar nessa questão.

---

Para melhorar produtividade, estou tentando dormir todo dia 00:00 e acordar as 8:00 em ponto. Além disso voltei a usar um [pomodoro timer](http://tomato-timer.com/), essa semana tem sido excelente depois dessas mudanças.

**Comment by victorserpac** [comment]
Posso dar uma sugestão @LFeh não muito relacionado ao tópico da Issue?

Estava lendo um artigo outro dia que dizia basicamente que dormir 7h30min era melhor do que dormir 8 horas. Fiquei intrigado e li o artigo em que dizia que basicamente nosso sono é constituído de ciclos de 90 minutos sendo cada ciclo constituído de estágios N1, N2, ... ([Link do artigo](http://noticias.terra.com.br/e-mesmo-necessario-dormir-oito-horas-por-noite,5e3c80e3b395088505dc886c41f8dcd5hxfopmh1.html))

Resolvi testar e notei que acordei mais disposto mesmo, e comecei a pedir para que amigos e familiares testassem essa teoria também. Todos realmente notaram que acordaram melhor, e pelo o que fala no artigo, é basicamente porque o ciclo do sono não é interrompido.

Bom, desculpa fugir um pouco do assunto mas acredito que essa informação possa ajudar :)

**Comment by felipefialho** [comment]
Mano, e faz muito sentido heim?

Vou fazer esse teste semana que vem. Tenho usado um app chamado "Sleep Better" que tenta aproveitar esses ciclos. Mas vou colocar pra me acordar 0:30 antes pra testar. 

Valeu por compartilhar.

**Comment by victorserpac** [comment]
Sim, esse app é bem da hora, já usei ele e gostei muito. Compartilhe depois se funcionou ou não essa teoria.

**Comment by dionarodrigues** [comment]
Então @victorserpac e @LFeh , estava lendo justamente sobre essa questão do sono dias atrás.

Descobri que em cada ciclo de 90 minutos, um adulto passa 80% aproximadamente em sono profundo e os outros 20% sonhando. Tendo isso em mente, podemos usar esses 20% em favor da nossa criatividade.  Essa fase dos sonhos é chamada de [REM](https://pt.wikipedia.org/wiki/REM_%28sono%29) e quanto mais próximos acordarmos dela, mais fácil será para lembrarmos dos sonhos que tivemos durante a noite.  Portanto, dormindo 7h30, teremos 5 ciclos, 5 momentos REM, 5 sonhos.

Mas o que isso tem a ver com criatividade?
Foi comprovado que sonhar nos ajuda a resolver problemas através de soluções criativas - já que nos sonhos nosso cérebro desliga a parte lógica e racional (as regras), nos fazendo ter pensamentos fora da caixa, fora do convencional. Quanto maior o tempo de sono REM, melhor a consolidação da nossa aprendizagem e maior a probabilidade de se ter insights depois de trabalhar na resolução de problemas. Foi assim que muitos cientistas chegaram a soluções que mudaram nossas vidas. =D

**Comment by mksanti** [comment]
Legas essa teoria dos ciclos. Não conhecia.

**Comment by bernardodiasc** [comment]
Massa. O assunto qualidade de vida tem tudo a ver com essa issue aqui kkkkk

Li um artigo que recomendava acordar, tomar um limão (com agua pq limao puro eh muito acido), depois 15 min de exercicio, ai meia hora depois do suco comer alguma coisa (cafe da manha, qq coisa eh melhor que nada), e finalmente somente depois disso tudo, ligar qualquer tela (celular, computador, tv, etc). Olha, 1 ano trabalhando em casa é suave, a coisa começa a ficar menos engraçada depois de 10 anos, entao o cuidado com a saude e evitar exageros eh extremamente importante. E sair de casa eh muito bom, muito recomendado pra quem trabalha em casa. :)

**Comment by victorserpac** [comment]
Que legal essa relação da criatividade com os ciclos @diogorodrigues. Muito interessante.

Realmente importante manter hábitos saudáveis @bernardodiasc. Não conhecia essa do limão...

**Comment by guilhermepontes** [comment]
cc: @mssantos dá seu depoimento hahah

**Comment by mssantos** [comment]
Comecei a trabalhar remotamente há sete meses e prefiro assim. 

Interessante o ponto de vista sobre evolução técnica que o @LFeh citou, no meu caso está sendo  diferente. Trabalhar dessa forma me deu oportunidade de trabalhar com pessoas muito fodas tecnicamente sem precisar me mudar de cidade.

Também tem a questão de ter o dia de trabalho mais previsível, não ter preocupação de trânsito/metrô/reunião de cliente, além de ter controle melhor sobre interrupção. Ou seja, você tem mais controle sobre seu dia.

Um drawback que as vezes me incomoda um pouco, mas que acontecia mais no início, é ser mais difícil de se desligar do trabalho. Fora isso, só sucesso.

**Comment by tcelestino** [comment]
Acho que o grande problema de trabalhar remoto, além dos citados acima, é a questão do ambiente que vai trabalhar. Eu hoje, não tenho muito interesse em ficar de home office, devido a qualidade do lugar que uso para trabalhar em casa. Sim, eu sou o culpado por isso. 

Parece mentira, mas ainda prefiro a estrutura que a empresa oferece. Tirando só a parte das cadeiras (não é @guilhermepontes!? hahahah).

**Comment by felipefialho** [comment]
@mssantos Você deu um exemplo legal.

De fato, se você mora em uma cidade menor, com menos profissionais de alto nível, você tem a mais oportunidade de trabalhar com profissionais assim e evoluir tecnicamente do que se trabalhasse alocado.

Não tinha pensado nisso.

**Comment by guilhermepontes** [comment]
@tcelestino hahahahahh

**Comment by andremw** [comment]
@diogorodrigues , interessante teu comentario. E eu ja passei por uma experiencia interessante com sonho, ha uns quatro anos atras. Eu tava ha uma semana quebrando a cabeça pra resolver um problema, e nada de conseguir. Um dia tava dormindo, comecei a sonhar com o problema e resolvi o problema no sonho. Logo que resolvi o problema eu acordei, peguei o notebook e tentei reproduzir a mesma coisa que tinha feito no sonho e.. deu certo! hahaha Sinistro!!

Eu to trabalhando remoto ha pouco mais de um ano. Concordo com todas as vantagens e desvantagens, mas algo que agora, depois de um ano longe da galera do escritorio, eu to sentindo mais, eh a falta de trocar ideia com o pessoal, de fazer mais amizade com a galera. Eu poderia ficar a vida toda do jeito que estou, mas pensar que eu poderia estar mais proximo da galera me deixa meio bolado.

**Comment by douglascorrea** [comment]
Eu estou há 1 anos e 5 meses trabalhando remoto de casa e gostou muito!

Concordo com o @LFeh que o foco é um dos principais problemas pois é "fácil" perde-lo.

Por outro lado, pelo menos pra mim, a evolução técnica foi absurda. Claro que o meu caso é um pouco diferente da maioria aqui pois ainda não sou Fulltime webdeveloper e minha ocupação principal é trabalhando com pacotes específicos da Oracle e, justamente por isso, ter um tempo maior para dedicar-me a estudar o que eu gosto (Desenvolvimento web) é fantástico.

Desde que estou trabalhando de casa estou conseguindo pegar alguns freelas de desenvolvimento web, me aprofundando mais em algumas tecnologias e estou a ponto de, finalmente, virar a chave para trabalhar somente com web (Fullstack). E tudo isso possibilitado pelo trabalho remoto, já que o dia a dia dentro de cliente de consultoria é uma correria absurda e não daria pra aprender o que conseguir aprender nos últimos meses.

**Comment by camilatigre** [comment]
Trabalho a 3 meses remota e, se tiver opção, não troco por alocado. Resolvi compartilhar um pouco da experiência, vai que eu ajudo alguém:

**O que é massa:** 
- Trabalho/Estudo quando me sinto mais produtiva
- Posso estar mais presente na vida da familia. O que no meu caso também inclui poder visitar minha familia minha familia/amigos em outro estado.
- Me ajudou a sair do comodismo. No alocado não senti tanto a falta de interagir com pessoas da área, ou pessoas, de modo geral. O Home-office me fez criar coragem pra ir em eventos conhecer gente e gente com interesses em comum, que é melhor ainda.
- Cuidar melhor da saúde. Consigo ir ao médico só pra fazer uns checkups sem ter que me sentir culpada de sair do trabalho por que quero cuidar da saúde. 
- Processos melhores definidos. Pra uma empresa deixar os funcionários como home-office acho que ela precisa ser no minimo organizada e isso influência diretamente no nosso trabalho.

**Com o que temos que tomar cuidado:**
- Seu horário de sono. Eu acabei desregulando o meu. Acabo dormindo como o @rkb81 falou, até as 14h, e a disposição física/mental as vezes fica zuada.
- Cuidar melhor da saúde. Sim, por que sem sair de casa você evita alguns virus mas também não pega sol, por exemplo. haha 
- Minha familia brinca que eu passo o dia no facebook e vendo série mas já conversei com outros home-offices e parece que tem gente que leva isso a sério. Desde entrar no quarto/escritório várias vezes à pedir/chamar pra resolver coisas seja em casa ou fora, são muitas as formas que podem te atrapalhar.

Tentei deixar mais claro e real os exemplos de vantagens e desvantagens que já citaram mas também sem repetir muito. Espero que eu tenha conseguido ((:

**Comment by fakenickels** [comment]
Como a @camilatigre falou tb, a família às vezes não leva você a sério. Te pede pra fazer coisas no horário do trabalho, chega visita na sua casa, fica no seu pé, é muito foda pq te tira do momento em que o foco tá mais "engatilhado". Mas tirando esse e outros pontos pra mim, trabalhar em casa é tudo de bom.

**Comment by kazzkiq** [comment]
Concordo com a @camilatigre e o @grsabreu, é muito difícil ensinar a família a respeitar seu "momento de trabalho" quando você é home-office.

E o pior é que esse tipo de comportamento "inoportuno" acaba fazendo você trocar a noite pelo dia, o que deve fazer um mal danado pro organismo (eu pelo menos fico sempre com aparência de cansado e com olheiras mesmo dormindo mais de 8 horas de dia).

Eu acho que nesse sentido, a melhor forma de conciliar o home-office com a família é montar um cômodo afastado e reservado apenas para trabalho. Meu ex-colega fazia isso: Acordava, tomava café com a familia e ia pro escritório trabalhar onde todo mundo sabia que não podia nem bater na porta enquanto ela estivesse fechada. Regra da casa. Essa foi a forma que ele encontrou pra conseguir fazer os filhos pequenos respeitarem o "trabalho do papai", mesmo estando em casa.

Uma outra coisa que pode ajudar (tanto na interferência familiar, quanto na própria disciplina do profissional) é trocar até de roupa na hora que for começar a trabalhar. Claro, não precisa ser um terno ou um vestido executivo, apenas uma roupa que não seja "aquelas de ficar em casa", o pijama ou algo do tipo. Isso pode parecer bobagem, mas o fato de acordar, colocar uma roupa mais social e ir pro ambiente de trabalho (em casa) gera um impacto psicológico muito forte, o que ajuda inclusive na produtividade e no respeito do seu espaço pela família. Pelo menos foi assim comigo.

**Comment by luisfmsouza** [comment]
Faz 1 mês que to trabalhando full remoto. E uma dificuldade que eu já percebi, é quando tenho que estudar (independente do tema) depois do expediente. 

Principalmente porque quando sento pra estudar, é exatamente no mesmo ambiente em que trabalho o dia todo. Não que o ambiente seja ruim, mas como já passei 8h do meu dia lá, fica um pouco complicado.

Como vocês tem feito?

**Comment by felipefialho** [comment]
@luisfmsouza Não tenho problemas com isso.

Mas já pensou em estudar no quintal da casa ou condomínio do prédio?

**Comment by WagnerMoreira** [comment]
Faz 1 ano que eu estou remoto, e além de todos os pontos positivos citados no primeiro post, eu acrescentaria:
- **Mais tempo livre pra viver**
  Dependendo do local onde você mora, geralmente vai perder pelomenos duas horas do seu dia no trajeto ida e volta ao trabalho, que te ocasiona cansaço e muito desgaste as vezes, tempo que você poderia usar para dormir melhor ou fazer alguma atividade do seu gosto, ler, academia, etc
- **A tranquilidade do ambiente**
  em alguns lugares onde trabalhei o barulho com muita gente nas salas atrapalhava **muito** a concentração
- **Você pode fazer seu próprio escritório**
  Um escritório com aquele esquema pra trabalhar em pé, e ir evoluindo a sua qualidade de vida e de trabalho ao mesmo tempo
- **Economizar com gastos com alimentação e ter uma alimentação mais saudável**
  A sua cozinha está bem ali do lado, pode parar pra fazer exercícios a qualquer hora, tomar banho pra relaxar quando você está resolvendo um problema muito complicado e coisas assim que você nunca pode fazer alocado

Nas desvantagens ali considero a parte de mais facilidade de perder o foco, nos primeiros dias eu tinha, mas depois eu virei o jogo e consigo me concentrar muito melhor sozinho ouvindo uma musica ou em silêncio total, diferente de alocado onde você corre o risco de ficarem te pingando toda hora tirando sua atenção e destruindo seu foco

E acrescentaria a parte de que raramente você vai encontrar um remoto CLT, o PJ pode acabar sendo muito furado dependendo do que você pretende fazer, considerando toda aquela coisa de que você não tem beneficio nenhum nem mesmo aumento anual, e vai ficar a mercê da boa vontade da empresa, sabemos que isso pode ser suicídio.

**Comment by luisfmsouza** [comment]
@LFeh já fiz algumas tentativas frustradas, pq durante a noite é quando o resto do pessoal já está por aqui.

O que eu vou testar nas próximas semanas é estudar de manhã, antes de "entrar no trabalho".

**Comment by rubenmarcus** [comment]
e nomadismo? Alguem ja conseguiu trabalhar que seja 3 meses, se deslocando pra outros paises?

vi 2 issues sobre o assunto aqui no forum, mas não falam do trabalho em si.

E os sites que vejo falando, parecem mais um conto de fábulas. Não gosto desses sites de "tendências" , Hypeness, Nomades Digitais e tantos outros, que tentam vender uma versão romantica e glamourizada de tudo.

Digo eu não quero saber os perrengues de ser nomade, porque eu já fiz algumas trips de mochilão e imagino que seja a mesma coisa.

Queria mais saber o caminho das pedras mesmo.

Pessoalmente, eu ja trabalhei como dev freelancer 2 anos em casa, e foi uma péssima experiencia, (não pra mim) , mas porque eu moro com a minha vó, e agência atrasava os pagamentos, eu ficava rolando divida com o banco, minha vó cobrava das contas, além claro de me cobrar de ir no mercado, lavar a louça, lavar a casa, atender a vizinha, ajudar com ela na televisão, qualquer coisa, quando eu ia ver ia só trabalhar as 6 horas da tarde e acabava parando as 6 da manhã..

Isso quando não extendia até as 9 da manhã, e dormi tipo até meio dia.

Mas o conforto é muito maior e a liberdade tambem.

Eu to reequipando meu quarto, pra transformar num estudio de arte,design e pra programar tambem. E quero chegar ao nivel de trabalhar remoto em algum tempo.

E mais pra frente de ter a liberdade de viajar pelo menos uns 3 meses ao ano, trabalhando...

Não sei se tem mais alguem aqui assim. Por mais que você consiga,conforto num bairro legal da cidade que você trabalha, a sensação de viajar, ver lugares novos e pessoas novas, conhecer histórias, desafios, etc etc, se sentir meio perdido por um momento, e quando estiver se familiarizando, você ir pra outro lugar, é um negocio meio viciante, digo demais até.

Enfim, desculpa ai o desabafo, se tiverem artigos ou histórias sobre nomadismo seja com design,web,artes e etc,me passem.

Valeu

**Comment by rubenmarcus** [comment]
btw, tem um local que funciona como retiro budista em Minas Gerais, em Extrema (Retiro Tao Tien), com comida vegetariana, local para meditação, cursos de pintura , cachoeira,etc. Que a diaria é 80 reais/dia.

Fiz um retiro no Carnaval de 2015 lá, e vi alguns profissionais de TI, trabalhando 6 horas por dia, e meditando,curtindo a natureza,tendo uma vida mais "saudavel", do que essa de pegar comida em fastfood, mercadinho,busão lotado.

Fora a vista de lá que é animal 

![Vista](https://static.wixstatic.com/media/5b884b_20e12ba024b148e8a8046164a9f67993.jpg/v1/fill/w_960,h_539,al_c,q_90/5b884b_20e12ba024b148e8a8046164a9f67993.jpg)

Digo, deve ser bem melhor trabalhar com uma vista dessas, do que casas e prédios, igual na minha casa , ou fileiras de mesas e computador (igual na empresa que trabalho)

**Comment by bernardodiasc** [comment]
Pro trabalho remoto funcionar, primordialmente 2 coisas devem estar bem alinhadas: a capacidade de haver um bom trabalho por parte do contratado e do contratante. O trabalho remoto e a vida nômade não são coisas inventadas nesse século, são coisas muito antigas. O que está havendo agora, especialmente nessa nossa área (que também não tem nada de antiga), é a possibilidade maior de levar o escritório inteiro consigo. É interessante ressaltar isso pra poder expandir as ideias. 

Não sei se existe 1 caminho das pedras, visto que a ideia é ter todos os caminhos que quiser trilhar e continuar obtendo sustento durante a jornada, seja ela qual for. Enquanto contratado, minha sugestão é adotar a postura de empreendedor, chefe de si, independente e proeficiente. Buscar trabalhar com contratantes que entendem o que isso tudo significa e que reconhecem os benefícios, mesmo que acompanhados dos pontos negativos.

Existem diversos motivos para as pessoas ou empresas procurarem profissionais remotos, mas não se engane, querer ser nômade não é argumento para conseguir trabalho remoto, nem de perto, business is business. O que vejo como sendo o maior motivador para esse tipo de contratação, são redução de custo e escassez de mão de obra qualificada local, sendo que as vezes nem é escasso e sim caro, reafirmando o fator redução de custo. Pra uma empresa, ter o profissional junto é muito vantajoso, mas aumenta os custos de diversos modos, transporte, infraestrutura, regulamentações, salário! e etc... Por exemplo, um profissional qualificado localizado em Sampa pode precisar de um salário bastante alto pra ter uma vida digna, com o mesmo custo pode ser possível achar profissionais remotos ainda mais qualificados se estes forem de locais onde o custo de vida é menor ou mesmo qualificação igual e um empregado muito mais satisfeito com a vida, como quiser imaginar.

Certamente nem tudo é sobre custos, contratantes que só tem isso em mente podem resultar em péssimas experiências, cuidado. O bom é procurar empresas que estão mais a frente do tempo e valorizam a diversidade cultural e experiências que uma pessoa de longe pode agregar ao time, entre outros motivadores. Algo que acredito é, se quer trabalhar remoto, deixa pra trás o modelo de assalariado, esquece o conforto do emprego, abraça o desafio e arca com as consequências, nem sempre boas, mas as vezes incríveis. E repito, nem tudo é bom mesmo, muita coisa não é nada fácil, mas acho que o que importa é quem manda na sua vida, você ou seu chefe? A liberdade tem seu preço e vale muito a pena pagar.

Lógico que nem tudo isso é regra, é apenas minha visão da coisa, funciona pra mim. Meus motivadores são: estar ao lado da minha esposa e filhos, morar onde eu quiser e trabalhar com projetos/empresas/times incríveis, e super importante, decidir meus horários de trabalho. :)

**Comment by rubenmarcus** [comment]
Entendi. Mas pra isso você tem que ser um profissional já reconhecido, ou com skills muito altas ou não? Ou algum cara que deu certo fazendo esses blogs/sites de tendencia que tem milhoes de views ou não?
Ou algum empreendedor wordpress com 100 clientes que deu sucesso??

Digo isso, Porque eu moro em SP, e aqui o custo de vida é muito alto, comparado ao que eu ganho, todo dia 5, a grana cai e ja vai embora, está complicado.

Comecei a fuçar no upwork e nos sites de freelance que pagam em dolar e euro a maioria é de 20 a 100 dolares no maximo e tem muita gente barganhando. de 100 trabalhos voce acha 1 de 700 dolares para cima.

O que eu to pensando em fazer pra conseguir essa independencia profissional e financeira é estudar mesmo. Ja estou fazendo isso, mas vou fazer mais.. É o unico caminho que eu vejo.

Eu digo com o conhecimento que eu tenho hoje, eu acho que dava pra ganhar no minimo mais meio salario numa empresa e mais um salario com freelas. Mas eu sou péssimo em negociar.
E tambem tenho problema com horários. 

Quando fiz mochilão em 2014 eu trabalhei pra clientes do Brasil, mas meus clientes são muito ruins, o cara cancelou o contrato comigo lá na Asia, e eu tive que me endividar no cartão pra pagar a viagem. Outro cliente que eu tinha ele me pagava conforme a demanda que era muito pouca, tipo fazer um hotsite a cada 2 meses, isso não custeava nem 2 semanas da minha viagem.

Enfim valeu o relato! @bernardodiasc

**Comment by bernardodiasc** [comment]
Se eu soubesse de tudo que passei antes de tentar, talvez pensasse duas vezes. rsrsrs Vou compartilhar humildemente mais um pouco da minha experiência, tem muita dificuldade que talvez não estejam claras nos blogs hypes mesmo. É bom saber das tretas, não para desanimar, mas para ter uma perspectiva realista diante das frustrações e lidar com inteligência para superar.

Comecei sendo bem desconhecido, quase um "sobrinho", me preocupei em entregar o melhor trabalho e atendimento possível, não importa o que custasse pra mim (muitas horas extras não remuneradas, mas SEMPRE entregava). Nunca precisei divulgar meu trabalho, trabalhei intensamente pra fazer renome e as pessoas me procuravam por indicações, frequentemente visitava os clientes (quando eram clientes maiores e próximos de onde eu morava), confiei nessa estratégia e deu muito certo, de indicação por indicação, tive momentos de mais trabalho do que conseguia dar conta, e fui tentando muitas estratégias diferentes no decorrer. Isso tem uns 10 anos já, foi mais ou menos como comecei e vivi por muitos anos. Ainda sou muito desconhecido, mas sei que se precisar posso procurar meus antigos clientes e com certeza os antigos bons relacionamentos e boas parcerias ainda significariam portas abertas. Somos todos pessoas antes de tudo!

Veja, nunca foi fácil pra mim. Pelo contrário. Tive que acumular muitos clientes e trabalhos simultâneos para poder ter rendimento, centenas (senão milhares) de madrugadas trabalhadas, etc. Como disse antes, acho que não existe uma receita pra ter sucesso nesse estilo de vida, o que precisa é ter postura empreendedora, comprometimento e muita coragem. A estratégia que mais me ajudou foi ter clientes fixos (e confiáveis), pagamentos previsíveis em datas certas todo mês, valores que variavam de 500 a 3 salários mínimos por cliente, somados aos esporádicos. Muito trabalho, muito jogo de cintura e muita pressão.

Sempre morei em cidades pequenas, onde o custo de vida é menor, isso conta, pois o rendimento de uma empresa de um homem só nem sempre é grande coisa. Hoje em dia trabalho exclusivamente para uma empresa estrangeira há pouco mais de 3 anos, bastante estável, eles confiam em mim e eu neles, mas nenhum benefício trabalhista, hora trabalhada é hora paga, basicamente isso. Férias e coisas do tipo são resultado de como administro minhas finanças pessoais. 

Enfim, o que importa mesmo são os relacionamentos, quanto mais duradouro melhor. Faça muitos contatos mas foque na qualidade desses contatos, acho que isso ajuda muito. Durante todo esse tempo, trabalhei _in loco_ por alguns períodos (_freelancing_ em paralelo, sempre), e alguns bons contatos surgiram dessas experiências, afinal, trabalhar com as pessoas, em pessoa, é uma ótima oportunidade para fazer bons relacionamentos profissionais.

Honestamente, não sei como está o mercado hoje. A internet evoluiu muito rápido em muito pouco tempo. O produto web barateou de forma grotesca (mérito do Open Source, o qual sou profundamente grato). Alguns anos atrás quando me dei conta das mudanças direcionei meu foco para o Serviço ao invés do Produto. Essa é a experiência que carrego comigo, enquanto programador sou também consultor. Enquanto profissional também sou bom parceiro de equipe. E por ai vai. Somos pessoas, não máquinas. :)

**Comment by rubenmarcus** [comment]
@bernardodiasc  Muito legal seu relato,  da pauta pra um artigo!

**Comment by bernardodiasc** [comment]
> O que eu to pensando em fazer pra conseguir essa independencia profissional e financeira é estudar mesmo. Ja estou fazendo isso, mas vou fazer mais.. É o unico caminho que eu vejo.

100% de acordo. As vezes que a cinta apertou com força, estudei sem parar, aprendi coisas novas e em pouco tempo estava colhendo frutos usando as coisas que aprendi. Fiz isso ano passado e garanti a continuidade do meu contrato vide que a empresa contratante migrou radicalmente de tecnologia, e quando fez isso, eu estava preparado. :)

**Comment by robertolealinvestidor** [comment]
Olá, já faz 2 anos que trabalho remotamente. No início foi muito dificil romber a barreira de entrada para finalmente ganhar dinheiro pela internet. Os primeiros clientes foram muito difíceis, trabalhei por muito pouco, muito pouco mesmo. Mas com o tempo fui ganhando espaço e visibilidade e as coisas começaram a melhorar. Acho que foi justamente nesse ponto, quando comecei a ganhar bem, que perdi o animo pelo trabalho remoto. Passo muito tempo sozinho, isso é ruim. O esforço para conquistar um cliente também não é pequeno. O freelancer precisa estar sempre motivado e acreditando em seu potencial para poder progredir. Acredito que fazer o que gosta é fundamental para ter sucesso no trabalho remoto. 
Estou buscando novos conhecimentos e novas áreas para me manter motivado. Agora tenho mergulhado sobre investimento. Sempre tive dificuldades com o dinheiro, em administrar meu dinheiro, em fazer render meus rendimentos. Realmente estou gostando dessa área. Semana passada encontrei um site muito bom para iniciantes: [xbinop](https://xbinop.com/br/)
Para quem nunca ouviu falar sobre [opções binárias](https://xbinop.com/br/estrategia-opcoes-binarias/), sugiro dar uma olhada.

**Comment by lucascaton** [comment]
Massa!

Eu escrevi um artigo sobre isso também há uns meses:
[Os desafios de trabalhar remotamente](https://www.lucascaton.com.br/2018/03/27/os-desafios-de-trabalhar-remotamente/)
